ibm/ibm-how-to/launch-ubuntu-instances.rst

```diff
@@ -0,0 +1,140 @@
+Launch an Ubuntu Virtual Server Instance (VSI) on IBM Cloud VPC infrastructure
+==============================================================================
+
+This how-to guide will give instructions for launching a Virtual Server Instance (VSI) on IBM Cloud using Ubuntu.
```

Comment:
Also wouldn't mind seeing links to IBM docs here, possibly for something explaining details about what a VSI is, since that is an IBM-unique term